workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -243,7 +243,9 @@ def get_manager_version(client):
     return ManagerVersion(client.manager.get_version()['version'])
 
 
-def get_tenants_list():
+def get_tenants_list(version):
+    if version < snapshot_constants.V_4_0_0:
```

Comment:
I don't think anyone anywhere is on anything earlier than 4.3, so we can probably just drop this conditional.